tests/integration-all/api-gateway/tests.js

```diff
@@ -12,7 +12,8 @@ const { createRestApi, deleteRestApi, getResources } = require('../../utils/api-
 
 const CF = new AWS.CloudFormation({ region });
 
-describe('AWS - API Gateway Integration Test', () => {
+describe('AWS - API Gateway Integration Test', function() {
+  this.timeout(1000 * 60 * 10); // Involves time-taking deploys
```

Comment:
Is there any way to set the timeout globally for all integration tests?

Reply:
There's nothing in Mocha that allows to set directory specific settings, and I don't think it's good to have it set to this value globally (e.g. that's how it was set with Jest)

Still I think it's not that big drawback, also timeout across integration tests may differ. Here we deal with CF deploys which are very long, but in case of packaging where there are only SDK calls such long timeout is not needed.